var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -14,15 +14,18 @@ class PyPyprecice(PythonPackage):
 
     homepage = "https://precice.org"
     git = "https://github.com/precice/python-bindings.git"
-    url = "https://github.com/precice/python-bindings/archive/v2.4.0.0.tar.gz"
+    url = "https://github.com/precice/python-bindings/archive/v3.1.1.tar.gz"
     maintainers("ajaust", "BenjaminRodenberg", "IshaanDesai")
 
-    license("LGPL-3.0")
-
     # Always prefer final version of release candidate
     version("develop", branch="develop")
+    version("3.1.1", sha256="50a0f1cbdb8fc362c22d316151c0e757958ff136a094e63b9b82d045d01d19c7")
```

Comment:
Couldn't verify the checksum for `py-pyprecice@=3.1.1`.
```
> spack checksum py-pyprecice 3.1.1
==> Found 1 version of py-pyprecice
==> Fetching https://github.com/precice/python-bindings/archive/v3.1.1.tar.gz

    version("3.1.1", sha256="112fccfbb223eb8df341221531f89943a14f75ee1e44dfb3276ebb38f21047de")
```

Reply:
I now also get the checksum `112fccfbb223eb8df341221531f89943a14f75ee1e44dfb3276ebb38f21047de`. I will correct it.

```suggestion
    version("3.1.1", sha256="112fccfbb223eb8df341221531f89943a14f75ee1e44dfb3276ebb38f21047de")
```

I assume that I got the checksum `50a0f1c` before we pushed the "post-tag bump" or there was some caching happening on github.

To provide some context here: Due to the system that we use for providing the version of the package via `precice.__version__` the checksum changes once after the release when the main branch does not point to the release anymore (usually when the next version is released). We make sure that the checksum remains the same by pushing a "post-tag bump" to the main branch directly after releasing (see https://github.com/precice/python-bindings/commit/14f1e0109935c56302dae5facad3740241248164 for https://github.com/precice/python-bindings/releases/tag/v3.1.1). See https://github.com/precice/python-bindings/issues/109 and https://github.com/python-versioneer/python-versioneer/issues/217.

I don't think it is possible to reproduce the checksum `50a0f1c` without force pushing to our main branch. So it is hard to do forensics and get 100% clarity.